### PR TITLE
Initial commit of readme and planning eval framework skeleton.

### PR DESCRIPTION
**Pull request by matthewclegg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,148 @@
...
+  -npublishers=2 \
+  -cost_model=fixed \
+  -frequencies=heterogeneous:3:5 \
+  -prices=0.05,0.06,0.07 \
+  -overlap=independent
+  -output=$DIR/data_design
+```
+
+In this example, we will use the sample experimental design in the file
+`examples/experimental_designs/goerg_gamma_poisson.py`. This compares Goerg's
```
Comment (kungfucraig) [inline README.md:115]: Maybe link the paper and use Georg's last name?

**Review by kungfucraig — APPROVED** [review]

**Review by uakyol — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1 +1,148 @@
...
+`data_generators/synthetic_data_generator.py`. Alternatively, if you have other
+data available in your organization, you can construct your own data design. An
+example data design can be found in the directory `examples/data_designs`.
+
+An Experimental Design specifies the models that will be evaluated and the
+parameters that will be used for evaluation. An Experimental Design is given by
+a Python class. An Experimental Design is subdivided into Experiments, which are
+further broken down into Trials. An Experiment consists of a collection of
+models and model parameters that are run against a single Dataset. A Trial
+consists of simulating the outcome of one model using one Dataset. For an
```
Comment (jiayu-google) [inline README.md:56]: The outcome of one model using one Dataset.

Perhaps + under one set of experimental parameters (e.g., number of training points, total privacy budget). I feel that experimental parameters are different from model parameters.

**Review by jiayu-google — APPROVED** [review]
Thank you Matthew! The readme is concise and to the point.

File: README.md (hunk 1 above)
Comment (matthewclegg) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -1 +1,148 @@
...
+## Overview
+
+This repo contains the code for the Planning Evaluation Framework, which
+evaluates planning models on both synthetic and real data. A planning model
+predicts the effectiveness of an advertising campaign that is spread across
+multiple publishers, by estimating the number of people that would be reached at
+various frequencies as a function of the amount spent with each publisher. The
+purpose of the Planning Evaluation Framework is to
+
+*   Provide a means for identifying the best candidate planning models to be
```
[the remark this answers is not in the document]
Comment (matthewclegg) [reply README.md]: Yes, that's better.

File: README.md [hunk 4]
```diff
@@ -1 +1,148 @@
...
+
+### Workflow
+
+**Note: Much of the following is aspirational and will be completed over the
+course of the project**
+
+A typical workflow consists of the following steps:
+
+1.  Generate a Data Design.
+1.  Configure an Experimental Design.
```
[the remark this answers is not in the document]
Comment (matthewclegg) [reply README.md]: I've changed the indexing, but by the way, some people prefer lazy indexing.  The correct numbering is generated by the markup renderer.

File: README.md [hunk 5]
```diff
@@ -1 +1,148 @@
...
+defines a Dataset. A Dataset in turn consists of a collection of Publisher Data
+Files. A Publisher Data File simulates the impression viewing history of a
+single publisher. It is a CSV file, where each line is of the form
+
+```
+   vid, spend
+```
+
+where vid is a virtual user id, and spend is the amount that would have to be
+spent for this impression to be shown.
```
[the remark this answers is not in the document]
Comment (matthewclegg) [reply README.md]: min(total_spend).  I've inserted the word 'total' to help make this explicit.

File: README.md [hunk 6]
```diff
@@ -1 +1,148 @@
...
+
+A Data Design can be synthetic, or it can represent actual publisher data.
+Synthetic data designs can be generated using the program
+`data_generators/synthetic_data_generator.py`. Alternatively, if you have other
+data available in your organization, you can construct your own data design. An
+example data design can be found in the directory `examples/data_designs`.
+
+An Experimental Design specifies the models that will be evaluated and the
```
[the remark this answers is not in the document]
Comment (matthewclegg) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (matthewclegg) [reply README.md]: Let me know if you like this wording better.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jiayu-google) [reply README.md]: Good to know!

File: README.md (hunk 2 above)
Comment (jiayu-google) [reply README.md]: Yes I like it. Thanks!

**Review by mpatel24 — APPROVED** [review]

**Pull request merged**